test-end-to-end/android-maestro/run-maestro-test.sh

```diff
@@ -13,7 +13,8 @@ TESTPASS="testpass"
 WORKDIR=$(pwd)
 TEST=""
 SCRIPTDIR=$(realpath $(dirname $0))
-TESTAPK=$SCRIPTDIR/../../app-android-launcher/build/outputs/apk/release/app-android-launcher-release.apk
+TESTAPK=/home/ustadmobile/Testfolder/app-android/build/outputs/apk/release/app-android-release.apk
```

Comment:
I think we shouldn't have changes to Maestro on this branch (this pull request is for cypress). Also the above change uses an absolute file path that would only work on the laptop it was run on.